To confirm these dependencies, run in a fresh build tree:


ninja muse/components/CMakeFiles/components.dir/confmport.o

In file included from ../muse/components/confmport.cpp:48:
../muse/mplugins/midifilterimpl.h:28:10: fatal error:
ui_midifilter.h: No such file or directory


ninja muse/waveedit/CMakeFiles/waveedit.dir/wavecanvas.o

In file included from ../muse/waveedit/wavecanvas.cpp:72:
../muse/components/copy_on_write.h:26:10: fatal error:
ui_copy_on_write_base.h: No such file or directory


ninja muse/instruments/CMakeFiles/instruments.dir/editinstrument.o

In file included from ../muse/instruments/editinstrument.cpp:58:
../muse/components/editevent.h:26:10: fatal error:
ui_editnotedialogbase.h: No such file or directory


ninja muse/liste/CMakeFiles/liste.dir/listedit.o

In file included from ../muse/liste/listedit.cpp:37:
../muse/components/editevent.h:26:10: fatal error:
ui_editnotedialogbase.h: No such file or directory


ninja muse/mixer/CMakeFiles/mixer.dir/rack.o

In file included from ../muse/mixer/rack.cpp:49:
../muse/components/plugindialog.h:4:10: fatal error:
ui_plugindialogbase.h: No such file or directory


--- a/muse/components/CMakeLists.txt
+++ b/muse/components/CMakeLists.txt
@@ -343,4 +343,5 @@ set_target_properties( components
 target_link_libraries ( components
       ${QT_LIBRARIES}
+      mplugins
       widgets
       xml_module
--- a/muse/waveedit/CMakeLists.txt
+++ b/muse/waveedit/CMakeLists.txt
@@ -79,4 +79,5 @@ set_target_properties( waveedit
 target_link_libraries( waveedit
       ${QT_LIBRARIES}
+      components
       widgets
       )
--- a/muse/instruments/CMakeLists.txt
+++ b/muse/instruments/CMakeLists.txt
@@ -78,4 +78,5 @@ set_target_properties( instruments
 target_link_libraries ( instruments
       ${QT_LIBRARIES}
+      components
       icons
       widgets
--- a/muse/liste/CMakeLists.txt
+++ b/muse/liste/CMakeLists.txt
@@ -65,4 +65,5 @@ set_target_properties( liste
 target_link_libraries ( liste
       ${QT_LIBRARIES}
+      components
       awl
       widgets
--- a/muse/mixer/CMakeLists.txt
+++ b/muse/mixer/CMakeLists.txt
@@ -87,4 +87,5 @@ set_target_properties ( mixer
 target_link_libraries ( mixer
       ${QT_LIBRARIES}
+      components
       widgets
       )
